### PR TITLE
Fix argument mismatch and pydantic constraint error

### PR DESCRIPTION
**Pull request by MarkintoshZ**
Encountered some error that seems to be the result from a previous refactoring.
- Could not find argument `enabled_analysis` in `Acto`
- Pydantic cannot satisfy constraint for `steps`

@tylergu could you check if these fixes are correct?

### CODE REVIEW

File: acto/lib/operator_config.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ class DeployConfig(BaseModel, extra="forbid"):
     """Configuration for deploying the operator"""
     steps: List[DeployStep] = Field(
         description="Steps to deploy the operator",
-        min_length=1,)
+        min_items=1,)
```
Comment (tylergu) [inline acto/lib/operator_config.py:48]: Can you try to upgrade the pydantic version to the version in the pyproject yaml and try again? I think the min_items is deprecated in the old version, and the min_length is the one to go with

Comment (MarkintoshZ) [reply acto/lib/operator_config.py:48]: I see. You are right. `min_length` works after I upgraded the pydantic version.

**Comment by tylergu** [comment]
I think the bug_reproduction CI failed because Acto crashed during the post diff test, but most likely not due to this PR.

**Comment by tylergu** [comment]
I am taking a look

**Pull request merged**